.circleci/config.yml

```diff
@@ -272,27 +268,33 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/code
-      - run: |
-          # Only sync to S3 for stable tags.
-          if [[ "$CIRCLE_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-            sudo apt-get update --allow-releaseinfo-change && sudo apt-get -y -qq install awscli
-            cd packages/core
-            aws s3 sync storybook-static "s3://mayflower.digital.mass.gov/core/v/$CIRCLE_TAG" --delete
-            # Use cp instead of sync for root copy.  sync lists all keys and is too slow.
-            aws s3 cp --recursive storybook-static "s3://mayflower.digital.mass.gov/core"
-            aws configure set preview.cloudfront true
-            aws cloudfront create-invalidation --distribution-id $AWS_DISTRIBUTION_ID --paths '/core*'
-          fi
+      # Only sync to S3 for stable tags.
+      - when:
+          condition:
+            matches:
+              pattern: '^[0-9]+\.[0-9]+\.[0-9]+$'
+              value: << pipeline.git.tag >>
+          steps:
+            - aws-cli/setup:
+                role-arn: $AWS_DEPLOYMENT_ROLE_ARN
+            - run: |
+                cd packages/core
+                aws s3 sync storybook-static "s3://mayflower.digital.mass.gov/core/v/$CIRCLE_TAG" --delete --profile default
+                # Use cp instead of sync for root copy.  sync lists all keys and is too slow.
+                aws s3 cp --recursive storybook-static "s3://mayflower.digital.mass.gov/core"
```

Comment:
question, does the `--profile default` flag only needs to be set once for each job?  I'm seeing sometimes it's repeated in every step, for example in `react_build_storybook`. Just want to make sure that it works as expected without the flag in subsequent steps

Reply:
Good catch! It should always be set on every aws command. It should be good now :+1: